tests/components/enocean/test_binary_sensor.py

```diff
@@ -0,0 +1,66 @@
+"""Test EnOcean light implementation."""
+import unittest
+from unittest.mock import MagicMock, patch
+
+from enocean.protocol.packet import RadioPacket
+
+from homeassistant.components import enocean as enocean_component
+from homeassistant.components.enocean import binary_sensor
+from homeassistant.const import STATE_OFF
+
+from tests.common import get_test_home_assistant
+
+
+class TestEnOceanBinarySensor(unittest.TestCase):
```

Comment:
We prefer standalone pytest test functions for new tests. See the smartthings test suite for example.

Reply:
Tests removed